README.md

```diff
@@ -184,11 +184,13 @@ An example use case looks like:
     "exp": <unix timestamp now+5min>,
     "iss": "<auth provider>",
     "aud": "cachenator,
-    "action": "READ"
+    "action": "READ",
+    "bucket": "mybucket",
```

Comment:
Would be good to mention in the readme that bucket/prefix are required, as we check in code if the claims match our struct, compared to `iss` and `aud` which are optional and enabled through an extra flag.

A note on ignoring would also be useful, like setting `bucket: ""` and `prefix: ""` to allow everything.